modules/home/assets/js/components/external-links-section.js

```diff
@@ -0,0 +1,35 @@
+import { Box, Paper } from '@elementor/ui';
+import List from '@elementor/ui/List';
+import ListItemButton from '@elementor/ui/ListItemButton';
+import ListItemText from '@elementor/ui/ListItemText';
+import Divider from '@elementor/ui/Divider';
+
+const ExternalLinksSection = ( { ...props } ) => {
+	return (
+		<Paper elevation={ 0 } sx={ { px: 3 } }>
+			<List>
+				{
+					props.externalLinksData.map( ( item, index ) => {
+						return (
+							<>
+								<ListItemButton key={ index } href={ item.url } target="_blank" sx={ { '&:hover': { backgroundColor: 'initial' }, gap: 2, px: 0, py: 2 } }>
```

Comment:
If I understand correctly, it's recommended to avoid using array index as key, but something constant, like item.label (assuming they're all unique).
You can read about this [here](https://stackoverflow.com/questions/59517962/react-using-index-as-key-for-items-in-the-list)

Reply:
you're right, in this case it is not so critical, since we are not going to rerender this component. But fixed anyway.